Automattic/wp-calypso
# Move several GitHub issue templates to YAML based template.

#### Changes proposed in this Pull Request

This PR proposes to migrate nearly all of our issue templates to use the new interactive GitHub issue template format.

The following templates are migrated:
- flakey e2e report
- tooling request
- simple/atomic feature parity

Outstanding items (to be handled in its own PR):
- happiness bug report
- bug report

#### Testing instructions

Ensure that templates load properly [here](https://github.com/Automattic/wp-calypso/tree/add/github-templates/.github/ISSUE_TEMPLATE).

Related to #54565
